catalog/opensearch/catalog-opensearch-source/src/main/java/org/codice/ddf/opensearch/source/OpenSearchSource.java

```diff
@@ -1044,50 +1056,48 @@ public PointRadius getPointRadius() {
   @Nullable
   protected SpatialSearch createCombinedSpatialSearch(
       final Queue<PointRadius> pointRadiusSearches, final Queue<Geometry> geometrySearches) {
-    final boolean filterContainedSomePointRadiusCriteria =
-        CollectionUtils.isNotEmpty(pointRadiusSearches);
-    final boolean filterContainedSomePolygonSearchCriteria =
-        CollectionUtils.isNotEmpty(geometrySearches);
-
-    if (filterContainedSomePointRadiusCriteria && filterContainedSomePolygonSearchCriteria) {
-      LOGGER.debug(
-          "Ignoring all spatial searches because combining spatial and point radius searches is not yet implemented.");
-      return null;
+    Geometry geometrySearch = null;
+    BoundingBox boundingBox = null;
+    PointRadius pointRadius = null;
+    SpatialSearch spatialSearch = null;
+
+    Queue<Geometry> combinedGeometrySearches = new LinkedList<>(geometrySearches);
```

Comment:
❓ Why is this collection of `Geometry`s a `Queue`? Why not a `Set`?

Reply:
changing to a set